chain/block.go

```diff
@@ -534,7 +534,11 @@ func (b *StatelessBlock) innerVerify(ctx context.Context) (merkledb.TrieView, er
 	processor.Prefetch(ctx, state)
 
 	// Process new transactions
-	unitsConsumed, surplusFee, results, stateChanges, stateOps, err := processor.Execute(ctx, ectx, r)
+	unitsConsumed, surplusFee, results, stateChanges, stateOps, err := processor.Execute(
```

Comment:
Can you remove these formatting diffs? I'd prefer to cleanup line length stuff in a separate PR.